### PR TITLE
Add display of task in state 'todo' of tickets and problems on homepage.

### PR DESCRIPTION
**Pull request by ddurieux**
Fix the fields in search engine, the completename for group to have 'my group' for groups in charge of the task and the name of the technician in charge of the task

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | yes

### CODE REVIEW

File: inc/commonitiltask.class.php [hunk 1]
```diff
@@ -1748,4 +1748,161 @@ function showFormMassiveAction() {
...
+               self::showVeryShort($ID, $itemtype);
+            }
+         }
+         echo "</table>";
+      }
+   }
+
+
+
+   static function showVeryShort($ID, $itemtype, $forcetab = '') {
```
Comment (trasher) [inline inc/commonitiltask.class.php]: missing phpdoc ;)

File: inc/commonitiltask.class.php [hunk 2]
```diff
@@ -1748,4 +1748,161 @@ function showFormMassiveAction() {
    }
 
 
+
+   /**
+    * @param $start
```
Comment (trasher) [inline inc/commonitiltask.class.php]: Incorrect phpdoc, see http://glpi-developer-documentation.readthedocs.io/en/master/codingstandards.html#comments

File: inc/commonitiltask.class.php [hunk 3]
```diff
@@ -1748,4 +1748,161 @@ function showFormMassiveAction() {
...
+    * @param $status             (default ''process)
+    * @param $showgrouptickets   (true by default)
+    */
+   static function showCentralList($start, $status = 'todo', $showgrouptickets = true) {
+      global $DB, $CFG_GLPI;
+
+      $query = "SELECT `".self::getTable()."`.`id` "
```
Comment (trasher) [inline inc/commonitiltask.class.php]: Use dbiterator?

File: inc/commonitiltask.class.php (hunk 2 above)
Comment (ddurieux) [inline inc/commonitiltask.class.php]: Oh yes you're right, it's not ok, I will modify it

File: inc/commonitiltask.class.php (hunk 3 above)
Comment (trasher) [inline inc/commonitiltask.class.php]: +1 for dbiterator; do not forget to remove the old query ;)

**Pull request merged**